### ISSUE TITLE
Sprint 3 – Scrum Meeting (15-12-2016) 

### ISSUE THREAD

**Issue by juzplayplay** [mewind/RSDSupermarketPOS#17]
Hi guys! Can we now start our meeting today?

**Comment by lkc442012** [comment]
Okay. Let me explain a bit about the work i done yesterday. Until now, I have already finish the product list in cart, calculation of subtotal, cancel function and calculation of discount rate based on user's log in. All these functions can work without getting error.

**Comment by juzplayplay** [comment]
Wow! That is great! So now can we proceed to next step? Do you guys have any idea?

**Comment by mewind** [comment]
Since the system is able to calculate the subtotal of the product selected and the discount rate for different type of users, maybe now we can focus on the calculation of grand total.

**Comment by huylin25** [comment]
I agree with Sin Nee. We should include the calculation the total for tax(GST), discount and subtotal.

**Comment by juzplayplay** [comment]
Okay. Maybe we should also display the grand total to the user after calculate all the cost that the user require to pay.

**Comment by mewind** [comment]
Yes. It should display besides the product list in cart in the payment page to allow the user to have a clear view about the products they purchased.

**Comment by lkc442012** [comment]
So the grand total will display the cost that adding subtotal and tax together but removing the discount price?

**Comment by huylin25** [comment]
Yes, you are correct.

**Comment by juzplayplay** [comment]
Kar Cheng, when will you are able to finish it?

**Comment by lkc442012** [comment]
This part seems like nothing much to do.... Maybe I can finish it by today.

**Comment by juzplayplay** [comment]
Alright, so today's task is just to complete the calculation of grand total?

**Comment by mewind** [comment]
Yaa, since the payment part is almost complete.

**Comment by juzplayplay** [comment]
Okay, then we can dismiss our meeting early. Kar Cheng, if you need any help just tell us.

**Comment by lkc442012** [comment]
Alright, thank you very much.

**Comment by huylin25** [comment]
Welcome!
